generator.py

```diff
@@ -139,6 +141,10 @@ def main():
             elif section == "SYMLINK_RULES":
                 print(f"LN\t{spec['src']} → {spec['dest']}")
                 symlink(spec["src"], spec["dest"])
+                # Remove dead symlinks
+                if not spec["dest"].exists():
+                    print(f"WARN: {spec['dest']} is a dead symlink, removed.")
+                    unlink(spec["dest"])
```

Comment:
This should be done after all symlinks are already processed, to detect chain symlinks correctly.

Reply:
Well. Dead symlinks are covered because symlinks should be created after all other files are copied or generated by distgen. That's the reason why the symlink section is the last one in the manifest config. Going one level deeper, if you want to create a symlink pointing to symlink, you just have to make sure that they're in the manifest config file in the correct order.
We don't need that functionality now and if you use it that way and without the correct order, the generator gives you a warning about it so you'd know what is happening. I think that's enough for now.

Reply:
Sounds good.